app/keepers/keepers.go

```diff
@@ -443,12 +456,15 @@ func NewAppKeeper(
 		pfmrouterkeeper.DefaultRefundTransferPacketTimeoutTimestamp,
 	)
 	transferStack = ratelimit.NewIBCMiddleware(appKeepers.RatelimitKeeper, transferStack)
+	transferStack = ibcfee.NewIBCMiddleware(transferStack, appKeepers.IBCFeeKeeper)
 
 	// Create ICAHost Stack
 	var icaHostStack porttypes.IBCModule = icahost.NewIBCModule(appKeepers.ICAHostKeeper)
+	icaHostStack = ibcfee.NewIBCMiddleware(icaHostStack, appKeepers.IBCFeeKeeper)
 
 	// Create Interchain Accounts Controller Stack
```

Comment:
```suggestion
	// Create Interchain Accounts Controller Stack
    // - core IBC  
    // - ibcfee  
    // - ICAController
```

Reply:
This got duplicated, it's the same as the comment above